src/moments.jl

```diff
@@ -58,200 +69,263 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
 The weights are assumed to be frequency weights, also called case weights.
 """
-mean_and_var(A::RealArray) = (m = mean(A); (m, varm(A, m)))
+function mean_and_var(A::RealArray; corrected=true)
+    m = mean(A)
+    v = varm(A, m; corrected=corrected)
+    m, v
+end
 
 """
-    mean_and_std(x, [wv::WeightVec], [dim]) -> (mean, std)
+    mean_and_std(x, [wv::AbstractWeights], [dim]) -> (mean, std)
 
 Return the mean and standard deviation of a real-valued array `x`, optionally
 over a dimension `dim`, as a tuple. A weighting vector `wv` can be specified
 to weight the estimates. The weights are assumed to be frequency weights, also
 called case weights.
 """
-mean_and_std(A::RealArray) = (m = mean(A); (m, stdm(A, m)))
+function mean_and_std(A::RealArray; corrected=true)
+    m = mean(A)
+    s = stdm(A, m; corrected=corrected)
+    m, s
+end
 
-mean_and_var(A::RealArray, wv::WeightVec) = (m = mean(A, wv); (m, varm(A, wv, m)))
-mean_and_std(A::RealArray, wv::WeightVec) = (m = mean(A, wv); (m, stdm(A, wv, m)))
+function mean_and_var(A::RealArray, wv::AbstractWeights; corrected=true)
+    m = mean(A, wv)
+    v = varm(A, wv, m; corrected=corrected)
+    m, v
+end
 
-mean_and_var(A::RealArray, dim::Int) = (m = mean(A, dim); (m, varm(A, m, dim)))
-mean_and_std(A::RealArray, dim::Int) = (m = mean(A, dim); (m, stdm(A, m, dim)))
+function mean_and_std(A::RealArray, wv::AbstractWeights; corrected=true)
+    m = mean(A, wv)
+    s = stdm(A, wv, m; corrected=corrected)
+    m, s
+end
+
+function mean_and_var(A::RealArray, dim::Int; corrected=true)
+    m = mean(A, dim)
+    v = varm(A, m, dim; corrected=corrected)
+    m, v
+end
 
-mean_and_var(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, varm(A, wv, m, dim)))
-mean_and_std(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, stdm(A, wv, m, dim)))
+function mean_and_std(A::RealArray, dim::Int; corrected=true)
+    m = mean(A, dim)
+    s = stdm(A, m, dim; corrected=corrected)
+    m, s
+end
+
+function mean_and_var(A::RealArray, wv::AbstractWeights, dim::Int; corrected=true)
+    m = mean(A, wv, dim)
+    v = varm(A, wv, m, dim; corrected=corrected)
+    m, v
+end
 
+function mean_and_std(A::RealArray, wv::AbstractWeights, dim::Int; corrected=true)
+    m = mean(A, wv, dim)
+    s = stdm(A, wv, m, dim; corrected=corrected)
+    m, s
+end
 
 ##### General central moment
 
-function _moment2(v::RealArray, m::Real)
+function _moment2(v::RealArray, m::Real; corrected=true)
     n = length(v)
     s = 0.0
     for i = 1:n
         @inbounds z = v[i] - m
         s += z * z
     end
-    s / n
+    s * bias(n, corrected)
 end
 
-function _moment2(v::RealArray, wv::WeightVec, m::Real)
+function _moment2(v::RealArray, wv::AbstractWeights, m::Real; corrected=true)
     n = length(v)
     s = 0.0
     w = values(wv)
     for i = 1:n
         @inbounds z = v[i] - m
         @inbounds s += (z * z) * w[i]
     end
-    s / sum(wv)
+
+    result = s * bias(wv, corrected)
```

Comment:
Agreed. Sorry, this is leftover from some debugging I was doing (removed the print statements, but forgot to reverse that).